code/game/objects/items/devices/lightreplacer.dm

```diff
@@ -65,12 +65,21 @@
 	// when we get this many shards, we get a free bulb.
 	var/shards_required = 4
 
+//monkestation edit  (using brped)
+/obj/item/lightreplacer/afterattack(obj/item/itemtarget, mob/living/user)
+	if(istype(itemtarget, /obj/machinery/light))
+		user.Beam(itemtarget, icon_state = "rped_upgrade", time = 5)
```

Comment:
Only play the beam and sound if at a distance, currently it does so even up close.